readthedocs/proxito/middleware.py

```diff
@@ -200,12 +200,7 @@ def process_request(self, request):  # noqa
         request.unresolved_domain = None
 
         skip = any(request.path.startswith(reverse(view)) for view in self.skip_views)
-        if (
-            skip
-            or not settings.USE_SUBDOMAIN
-            or "localhost" in request.get_host()
-            or "testserver" in request.get_host()
```

Comment:
These two checks are not really that useful, and they cause problems if your project has `localhost` in their name.

Reply:
Hrm, I feel like it might cause some issues in tests, but if everythings looks OK, I agree this is a better approach.